portfolio/src/main/webapp/style.css

```
@@ -16,6 +19,34 @@
   flex-direction: row;
   justify-content: space-between;
 }
+.flex-col {
+  display: flex;
+  flex-direction: column;
+}
+form{
```

[4.2.7](https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation): Separate rules by new lines.

```suggestion
}

form {
```

Here and elsewhere.